crates/core/c8y_smartrest/src/event.rs

```diff
@@ -0,0 +1,109 @@
+use thin_edge_json::event::ThinEdgeEvent;
+use time::{format_description::well_known::Rfc3339, OffsetDateTime};
```

Comment:
I'm glad you have chosen to use `time` crate instead of `chrono` and very slowly the Rust community is leaning towards the `time` crate. 

I have had a PR to convert all usage of `chrono` in the our codebase but it didn't go through as there are still some crates that use `chrono`.

My question is should we start using `time` from all the new code or for the time being keep using `chrono`?

Reply:
We started using this `time` crate instead of `chrono` when the alarms feature was implemented and that was a conscious decision to use `time` in our future code as we're eventually headed there. So, just continuing the same.

Reply:
Thanks, I didn't notice it there :)
In this case I maybe should revive the other PR to swap `chrono` for `time` where possible.